odrive_node/msg/ODriveStatus.msg

```diff
@@ -1,3 +1,5 @@
+std_msgs/Header header
+uint32 node_id
```

Comment:
Your PR description states that this node_id should serve to distinguish two or more ODrive sources. But the typical way to do this that I know of is via the topic path on which they are published:
```sh
ros2 topic echo /odrive_axis0/odrive_status
ros2 topic echo /odrive_axis1/odrive_status
...
```
Is there a scenario where this does not work?